City-of-Pori/pori-events
# PORI-231: Events base theme and header & footer theming.

Events base theme and header & footer theming.

To test:
1. first you need to have working pori-events local setup.
2. drush cim -y && drush cr
3. make sure pori_events theme is enabled.
4. go to /admin/structure/block/list/pori_events there you should have 3 blocks in footer region
5. you can provide content for the blocks in custom blocks tab in the block menu
6. go back to front page and confirm that there is a logo in header and footer, footer blocks should show the text.
7. Slogan is provided from template file hardcoded, since the text is too long for the actual slogan field in site settings. Slogan is hidden in mobile for now since there is no menu implemented yet, also language-switcher does not exist yet.
